landlab/components/overland_flow/generate_overland_flow_deAlmeida.py

```diff
@@ -368,7 +369,7 @@ def dt(self):
 
     @dt.setter
     def dt(self, dt):
-        assert dt > 0
+        assert dt > 0, "timestep dt must be positive"
```

Comment:
I would rather remove the *assert* here. Instead, how about:
```python
if dt <= 0:
    raise ValueError("time step must be positive")
```
*assert* statements should really only be used in debugging since they are only run when the Python ``__debug__`` variable is set to ``True``.